### ISSUE TITLE
Using Cloud GPU

### ISSUE THREAD

**Issue by vaibhavahuja** [log0/neural-style-painting#9]
How can i use floydhub/AWS for implementation of this? My PC would hang everytime i came on the step where it applies gradient descent for reducing the loss function.

**Comment by vaibhavahuja** [comment]
never mind. I did implement it on floydhub. If anyone wants to do that, just read the documentation properly once and you are good to go.
